src/scribe_data/wikidata/language_data_extraction/arabic/adverbs/query_adverbs_1.sparql

```diff
@@ -0,0 +1,21 @@
+# tool: scribe-data
+# All arabic (Q13955) adverbs (Q380057) and their forms.
+# Enter this query at https://query.wikidata.org/.
```

Comment:
nit - just a suggestion

Would it make sense to include a blurb in these headers that the files are being generated? With something along the lines of..
> Auto-generated by the `check_and_update_missing_query_forms.yaml` workflow.